config/config.go

```diff
@@ -0,0 +1,35 @@
+package config
+
+import (
+	"github.com/BurntSushi/toml"
+	log "github.com/Sirupsen/logrus"
+)
+
+// Config is the application-wide config
+var Config AppConfig
+
+func init() {
```

Comment:
Great question. http://stackoverflow.com/questions/24790175/when-is-the-init-function-in-go-golang-run